onnx2kerastl/reshape_layers.py

```diff
@@ -193,8 +204,11 @@ def target_layer(x):
                     flatten = keras.layers.Flatten(name=keras_name)
                     layers[node_name] = flatten(input_0)
                 else:
-                    reshape = keras.layers.Reshape(np.int32(input_1[1:]), name=keras_name)
-                    layers[node_name] = reshape(input_0)
+                    if abs(len(input_0.shape) - len(input_1)) == 1:  # keras reshape don't work
+                        layers[node_name] = tf.reshape(input_0, input_1, name=keras_name)
```

Comment:
Can you describe in what cases this if block will be used?

Reply:
Changed this to a new condition that considers batch changes only